open_cv_camera.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import time
+
+import cv2
+
+
+def main():
+    """This program is to take multiple images for AIEIO when an assigned key is pressed
+    --- You may adjust the amount of pictures you want for each press by changing the
+    assigned range :)
+    """
+
+    try:  # set emotion
+        emotion = sys.argv[1]
+    except IndexError:
+        emotion = "happy"
+        print("No emotion specified, set to default \"happy\".")
+    try:  # set number of pics to take each time
+        numpics = sys.argv[2]
+    except IndexError:
+        numpics = 150
+        print("Number of images not specified, set to default 150.")
+
+    path = os.path.join(os.getcwd(), 'images', emotion)
+    if os.path.isdir(path):
+        print("Directory was not created since %s already exists." % path)
+    else:
+        try:
+            os.mkdir(path)
+        except OSError:
+            print("Creation of the directory %s failed" % path)
+        else:
+            print("Successfully created the directory %s " % path)
+
+    try:  # sets image counter to avoid overwriting old images from max of target directory
+        img_counter = int(max(os.listdir(path)).split(".png")[0].split("_")[1]) + 1
```

Comment:
split this to more lines so that we understand it. 

also, access index of a list without checking it is a bad habit.